webapp-django/crashstats/base/tests/testbase.py

```diff
@@ -30,6 +31,8 @@ def setUp(self):
         # attacking first in
         # https://bugzilla.mozilla.org/show_bug.cgi?id=1188083
         classes_with_implementation = (
```

Comment:
Do you think that you will eventually be able to remove this hard-coded list, when all services have an implementation?